frontend/src/metabase/dashboard/reducers.js

```diff
@@ -515,6 +516,13 @@ const autoApplyFilters = handleActions(
   INITIAL_DASHBOARD_STATE.autoApplyFilters,
 );
 
+const theme = handleActions(
+  {
+    [SET_DISPLAY_THEME]: { next: (state, { payload }) => payload ?? "light" },
```

Comment:
Some areas in the code still seems to pass `null` or `undefined` here, so we coalesce to `light` for clarity.

Reply:
Could we use `createReducer` from `@reduxjs/toolkit` instead? It's the [preferred solution.](https://metaboat.slack.com/archives/C505ZNNH4/p1692634080082229?thread_ts=1692631233.986049&cid=C505ZNNH4)